docs/content/users/quickstart.md

```diff
@@ -607,6 +607,65 @@ The Laravel project type can be used for [Statamic](https://statamic.com/) like
     ddev launch /cp
     ```
 
+## Sulu
+
+```bash
+mkdir my-sulu-site && cd my-sulu-site
+ddev config --project-type=php --docroot=public
+ddev start
+ddev composer create sulu/skeleton
+```
+
+!!!tip "Multilingual"
+    To set up the administration interface for example in French alongside the default English, add the following snippet on top of `config/packages/sulu_admin.yaml` ([all available localizations](https://sulu.crowdin.com/sulusulu)):
+
+    ```bash
+    sulu_core:
+    locales:
+        en: English
+        fr: Francais
+    translations:
+        - en
+        - fr
+    ```
+
+    `ddev exec bin/console sulu:admin:download-language` downloads any set language. The only exceptions are English and German which are shipping with Sulu.
+
+The content management part of Sulu is built upon webspaces. Each webspace configures a content tree. Each content tree may contain translations for different locales. Create your default webspace configuration `mv config/webspaces/example.xml config/webspaces/my-sulu-site.xml` and adjust the values for `<name>` and `<key>` so that they are matching your project:
+
+```bash
+<?xml version="1.0" encoding="utf-8"?>
+<webspace xmlns="http://schemas.sulu.io/webspace/webspace"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://schemas.sulu.io/webspace/webspace http://schemas.sulu.io/webspace/webspace-1.1.xsd">
+    <!-- See: http://docs.sulu.io/en/latest/book/webspaces.html how to configure your webspace-->
+
+    <name>My Sulu CMS</name>
+    <key>my-sulu-cms</key>
+```
+
+!!!warning "Caution"
+    Changing the `<key>` for a webspace later on causes problems. It is recommended to decide on the value for the key before the database is build in the next step.
+
+The information for the database connection is set in the environment variable `DATABASE:URL`. For development, clone the `.env` file by `cp .env .env.local` and change the environment variable to:
+
+```bash
+DATABASE_URL="mysql://db:db@db:3306/db?charset=utf8mb4"
```

Comment:
All Symfony or even doctrine based applications should define the server_version else you run into unexpected errors:

```suggestion
```bash
APP_ENV=dev
DATABASE_URL="mysql://db:db@db:3306/db?serverVersion=8.0.32&charset=utf8mb4"
```

Reply:
as mentioned in one other comment i am not a developer and was following the recommendation by @rfay who raised 

> We don't want to put an explicit patch version of mysql in there, it will break often. If we used --database=mysql:8.0 above that does the job.

in https://github.com/ddev/ddev/pull/6256#discussion_r1625078826 . with the serverversion in the user would have to keep the version version updated in case the database image is getting updated in a new version of ddev?

Reply:
yes but a project normally should not get automatically a new minor version, as doctrine did in past behave differently based on minor versions (e.g. JSON was stored as TEXT in older mariadb/mysql versions) and if it does not match your "production" version you even could accidently create false databasemigrations which do work locally but not on online server.

Reply:
Database version is updated with DDEV versions; only the major version (like 8.0, 8.4 for MySQL, and 10.11, 11.4 for MariaDB, or `14` or `15` for PostgreSQL) is available. Maintaining a patch version in settings code won't work out in the long run (or the short run :) )

Reply:
Minor rewrite of @alexander-schranz suggestion, without MySQL patch version.

```suggestion
APP_ENV=dev
DATABASE_URL="mysql://db:db@db:3306/db?serverVersion=8.0&charset=utf8mb4"
```